decentraland/helpers/index.js

```diff
@@ -182,6 +180,18 @@ const loadConfig = () => {
   ConfigLoader.setConfig(tasitSdkConfig);
 };
 
+export const openURL = async url => {
```

Comment:
Any reason this was introduced into the helpers just now as opposed to at a different time? I can't see how it relates to this PR. Which is fine, just curious. Thanks!

Reply:
It isn't clear why (if it's a babel or a react issue), but there are some times that `ProviderFactory` wasn't properly configured (the ConfigLoader.setConfig was being executed after). I've faced that issue again when tried to refactoring the code to avoid config file name duplications.
For example: Inside the app, even config is being loaded on the App component, a contract instantiation isn't working out of a function (global var or class field).

Reply:
Interesting. Sounds like something is async that we're not awaiting properly?

Maybe we should have a config via a singleton class that is instantiated once and then is used in components via an import to then access it inside a function in that component?

Reply:
Actually I changed my mind, most likely isn't an async problem. But the second part of my message still could be worth trying.

Reply:
> Maybe we should have a config via a singleton class that is instantiated once and then is used in components via an import to then access it inside a function in that component?

Note: Not sure if I've understood your question.

We have the `ConfigLoader` class (singleton) that is doing that (is a singleton that is used by `ProviderFactory`). The code below is supposed to work and it is working on our `tasit-action` test suites.

```javascript
ConfigLoader.setConfig(config);
ProviderFactory.getProvider();
```
But on the `tasit` project, if I do that outside of a class, it doesn't work some times. Seems that babel during transpile is changing the order of these commands.

Reply:
Ah ok, that makes sense, right. So we already do what I was suggesting.

Hmm, interesting. Maybe we can find a way to detect this condition and to throw an error.

Reply:
Should we create an issue for this if there isn't one already?

Reply:
Yes, we can, I guess that could be solved with a properly module to deal with the config file instead of import it manually. So, should I
1) Create a new short term issue to deal with that problem related above
2) Add a note to the `TasitSDK` issue https://github.com/tasitlabs/TasitSDK/issues/184
3) Create a new `tasit` issue about using a properly config module
?

Reply:
1. It's working with the hack right now, right? Or is it not working reliably? If it's working with the hack, no short-term issue needed.
2. Sounds good.
3. Sounds good.

Reply:
1. Yes
2. Done
3. Done (https://github.com/tasitlabs/tasit/issues/278)

Reply:
Thanks! Resolving this